specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/examples/DeleteIdentity.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "api-version": "2021-03-07",
+    "content-type": "application/json",
+    "id": "8:acs:2dee53b4-368b-45b4-ab52-8493fb117652_00000005-14a2-493b-8a72-5a3a0d000081",
+    "endpoint": "https://my-resource.communication.azure.com"
```

Comment:
Should `endpoint` instead have the identities path and become `"https://my-resource.communication.azure.com/identities/"`? This applies to all examples

Reply:
No, it is on purpose changed for all examples https://teams.microsoft.com/l/message/19:9c97bce4d74c4fa5b6571e656ef71854@thread.skype/1609926761230?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1609926761230&teamName=Azure%20Communication%20Services&channelName=Docs&createdTime=1609926761230
![image](https://user-images.githubusercontent.com/17181592/105477875-c08f3480-5ca2-11eb-8c45-53470393027d.png)